rosdep/python.yaml

```diff
@@ -5786,6 +5786,10 @@ python3-psutil:
   rhel: ['python%{python3_pkgversion}-psutil']
   slackware: [psutil]
   ubuntu: [python3-psutil]
+python3-pyassimp:
+  debian: [python3-pyassimp]
```

Comment:
It looks like python3-assimp is only available on debian since buster.  Since we still support stretch for Melodic, I think we need to change this to:

```
  debian:
    '*': [python3-pyassimp]
    stretch: null
```